web/src/API.re

```diff
@@ -0,0 +1,42 @@
+open Relude.Globals;
+
+module Error = {
+  type t = ReludeFetch.Error.t(string);
+  let show = error => ReludeFetch.Error.show(a => a, error);
+  module Type = {
+    type nonrec t = t;
+  };
+};
+
+module Endpoint = {
+  type t =
+    | URLs;
+
+  let unwrap =
+    fun
+    | URLs => "/api/url/list";
+};
+
+module URLs = {
+  [@decco]
```

Comment:
This is an example `how to use decco`. 
basically, reason don't have a type for JSON, so we need `decode/encode` from json to reason type.